src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -473,31 +473,130 @@ function prepareConstSumQuestions() {
+ * must be satisfied for arriving at a valid distribution
+ * @param qnNum - question number
+ */
+function updateConstSumQnInstructions(qnNum) {
     let numOptions = 0;
     let points = parseInt($(`#constSumPoints-${qnNum}`).val(), 10);
+
+    const numRecipients = parseInt($(`[name="questionresponsetotal-${qnNum}"]`).val(), 10);
+
+    // Boolean value stating the type of CONSTSUM question (among recipients or options)
+    const distributeToRecipients = $(`#constSumToRecipients-${qnNum}`).val() === 'true';
+    /*
+     * Boolean value stating whether total points to be distributed
+     * is equal to the value of variable `points` or `points`x no. of options
+     */
+    const pointsPerOption = $(`#constSumPointsPerOption-${qnNum}`).val() === 'true';
+    /*
+     * Boolean value which is true when all recipients/options can not be given
+     * the same amount of points.
+     * This variable is being used mainly for backward compatibility
+     */
+    const forceUnevenDistribution = $(`#constSumUnevenDistribution-${qnNum}`).val() === 'true';
+
+    /*
+     * For "among recipients" type of question, consider no. of options
+     * to be equal to the no. of recipients. Otherwise, extract it from
+     * the hidden input element `#constSumNumOption`
+     */
+    if (distributeToRecipients) {
+        numOptions = numRecipients;
+    } else {
+        numOptions = parseInt($(`#constSumNumOption-${qnNum}`).val(), 10);
+    }
+
+    if (pointsPerOption) {
+        points *= numOptions;
+    }
+
+    // Specifies the type of distribution constraint selected by question creator
+    const constSumDistributePointsOptionSelected = $(`#constSumDistributePointsOptions-${qnNum}`).val();
+    // Specifies whether all recipients/options should be allocated different points
+    const forceAllUnevenDistribution = constSumDistributePointsOptionSelected
+            === ParamsNames.FEEDBACK_QUESTION_CONSTSUMALLUNEVENDISTRIBUTION;
+    // Specifies if all points can't be the same i.e. at least one allocation must differ from the rest
+    const forceSomeUnevenDistribution = constSumDistributePointsOptionSelected
+            === ParamsNames.FEEDBACK_QUESTION_CONSTSUMSOMEUNEVENDISTRIBUTION;
+
+    const instructionElement = $(`#constSumInstruction-${qnNum}`);
+    let instruction = '';
+    const infoIcon = '<span class="glyphicon glyphicon-info-sign padding-right-10px"></span>';
+
+    instruction += `${infoIcon} Total points distributed should add up to ${points}.<br>`;
+
+    // Check if allocating different points to just one recipient/option will be enough
+    if (forceSomeUnevenDistribution) {
+        instruction += `${infoIcon} At least one ${distributeToRecipients ? 'recipient' : 'option'} `
+                + 'should be allocated different number of points.<br>';
+    } else if (forceUnevenDistribution || forceAllUnevenDistribution) {
+        // Each recipient/option must be allocated different points otherwise
```

Comment:
i think this comment is unnecessary

Reply:
and this part is where the backward integration variable comes in. You shld explain it.